lib/parallel_tests/cli.rb

```diff
@@ -57,8 +57,8 @@ def execute_in_parallel(items, num_processes, options)
       Tempfile.open 'parallel_tests-lock' do |lock|
         ParallelTests.with_pid_file do
           simulate_output_for_ci options[:serialize_stdout] do
-            Parallel.map(items, in_threads: num_processes) do |item|
-              result = yield(item)
+            Parallel.map_with_index(items, in_threads: num_processes) do |item, index|
```

Comment:
If I understand all of this correctly, the `item` that gets yielded on the next line is the `group` in the `execute_in_parallel` call on line 84

If that's the case, I wonder if naming these to `groups`/`group` instead of `items`/`item` makes it easier to understand?

Reply:
execute_in_parallel is a generic method that does not always get groups passed
for example `execute_in_parallel(runs, runs.size, options)`
so afaik this should stay items